src/Futhark/CodeGen/Backends/SequentialWASM.hs

```diff
@@ -0,0 +1,66 @@
+-- | C code generator.  This module can convert a correct ImpCode
+-- program to an equivalent C program.  This C program is expected to
+-- be converted to WebAssembly, so we also produce the intended
+-- JavaScript wrapper.
+module Futhark.CodeGen.Backends.SequentialWASM
+  ( compileProg,
+    runServer,
+    GC.CParts (..),
+    GC.asLibrary,
+    GC.asExecutable,
+    GC.asServer,
+  )
+where
+
+--import Language.Futhark.Core (nameToString) -- TODO (see if might be useful later)
```

Comment:
Remove this line.

Reply:
Why did you not remove this?  Am I misreading the diff?

Reply:
Sorry that was an oversight